ci-operator/step-registry/ipi/conf/vsphere/zones/multisubnets/external-lb/ipi-conf-vsphere-zones-multisubnets-external-lb-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: ipi-conf-vsphere-zones-multisubnets-external-lb
+  from: upi-installer
+  commands: ipi-conf-vsphere-zones-multisubnets-external-lb-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
```

Comment:
Seem like we do not need these credentials in this step.

Reply:
ok, but how to judge whether to use these credentials or not?

Reply:
Searching `/var/run/vault/vsphere` in the commands.sh

Reply:
thanks.